csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -5187,6 +5187,8 @@ CSAF: https://domain.tld/security/csaf/provider-metadata.json
 CSAF: https://www.example.com/.well-known/csaf/provider-metadata.json
 ```
 
+> An issuing party MAY advertise more than one `provider-metadata.json` by adding multiple `CSAF` fields, e.g. in case of changes to the organizational structure through merges or aquisitions. It is strongly recommended NOT to do this. If it is inevitable, this SHOULD be a temporary solution. If one of the URLs fulfills requirement 9, this SHOULD be used as the first CSAF entry.
```

Comment:
MAY and SHOULD contradict themselves according to RFC2119. My suggestion is to remove the MAY and say something like _it is possible to advertise more than one ....but it SHOULD NOT be done and be removed as soon as possible._  Addition I think that the last hint should be a MUST like _If one of the URLs fulfills requirement 9, this MUST be used as the first CSAF entry._

Reply:
Having a MUST is IMHO usually a No-Go in an informal statement... See [f2e97a8](https://github.com/oasis-tcs/csaf/pull/430/commits/f2e97a86f6881e08915b689922fa9548640dfd1b) for my suggestion.

Reply:
If it is informal then let us use English spelling (lowercase may, should, …) and reduce the appealing parts?

Reply:
I did convert it into an formal one :see_no_evil: